packages/kit/src/core/make_package/index.js

```diff
@@ -89,11 +91,16 @@ export async function make_package(config, cwd = process.cwd()) {
 		JSON.stringify(package_pkg, null, '  ')
 	);
 
-	const project_readme = path.join(cwd, 'README.md');
-	const package_readme = path.join(cwd, config.kit.package.dir, 'README.md');
+	const whitelist = fs.readdirSync(cwd).filter((file) => {
+		return essential_files.some((name) => file.startsWith(name));
```

Comment:
I think letter case allignment needs to happen here. Else `readme.md` is filtered out, `startsWith` is case sensitive

Reply:
You're too quick! Was just about to push an update for this too